### PR TITLE
feat(oui-sidebar): add sidebar style

### PR DESCRIPTION
**Pull request by AxelPeter**
- Add style for oui-sidebar
- Add examples for oui-sidebar documentation

Linked to https://github.com/ovh-ux/ovh-ui-kit-documentation/pull/143

### CODE REVIEW

File: packages/oui-navbar/navbar.less [hunk 1]
```diff
@@ -406,7 +406,7 @@
       color: @oui-navbar-menu-title-font-color;
     }
 
-    // Bradcrumb for Header
```
Comment (FredericEspiau) [inline packages/oui-navbar/navbar.less]: https://goo.gl/images/zLqZcj

File: packages/oui-sidebar/README.md [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
+          Category 1
+        </button>
+      </li>
+      <li class="oui-sidebar-list__item" aria-haspopup="true" aria-expanded="false">
+        <button type="button" class="oui-sidebar-link">
+          Category 2
+        </button>
+      </li>
+      <li class="oui-sidebar-list__item">
+        <button class="oui-sidebar-link oui-sidebar-link_active" aria-haspopup="true" aria-expanded="true">
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: `type`

File: packages/oui-sidebar/README.md [hunk 3]
```diff
@@ -0,0 +1,689 @@
...
+          <header class="oui-sidebar-menu__header">
+            <div class="oui-sidebar-menu__back">
+              <button type="button" class="oui-button oui-button_link oui-button_icon-left">
+                <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
+              </button>
+            </div>
+            <span class="oui-sidebar-menu__breadcrumb">
+              Manager
+            </span>
+            <h3 class="oui-sidebar-menu__title">
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: No `h1` and `h2` above

File: packages/oui-sidebar/README.md [hunk 4]
```diff
@@ -0,0 +1,689 @@
...
+              Category 3
+            </h3>
+          </header>
+          <ul class="oui-sidebar-list">
+            <li class="oui-sidebar-list__item">
+              <button class="oui-sidebar-link oui-sidebar-link_active" aria-haspopup="true" aria-expanded="true">
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: `type`

File: packages/oui-sidebar/README.md [hunk 5]
```diff
@@ -0,0 +1,689 @@
...
+                Product 1
+              </button>
+              <!-- Level 2 -->
+              <div class="oui-sidebar-menu oui-sidebar-menu_fixed">
+                <header class="oui-sidebar-menu__header">
+                  <div class="oui-sidebar-menu__back">
+                    <button type="button" class="oui-button oui-button_link oui-button_icon-left">
+                      <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: Accessibility: how do blind people know what this button does ?

File: packages/oui-sidebar/README.md [hunk 6]
```diff
@@ -0,0 +1,689 @@
...
+          <ul class="oui-sidebar-list">
+            <li class="oui-sidebar-list__item">
+              <button class="oui-sidebar-link oui-sidebar-link_active" aria-haspopup="true" aria-expanded="false">
+                Product 1
+              </button>
+              <div class="oui-sidebar-menu oui-sidebar-menu_fixed">
+                <header class="oui-sidebar-menu__header">
+                  <div class="oui-sidebar-menu__back">
+                    <button type="button" class="oui-button oui-button_link oui-button_icon-left">
+                      <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: Accessibility: how do blind people know what this button does ?

File: packages/oui-sidebar/README.md [hunk 7]
```diff
@@ -0,0 +1,689 @@
...
+            <span class="oui-sidebar-menu__breadcrumb">
+              Manager
+            </span>
+            <h3 class="oui-sidebar-menu__title">
+              Category 3
+            </h3>
+          </header>
+          <ul class="oui-sidebar-list">
+            <li class="oui-sidebar-list__item">
+              <button class="oui-sidebar-link oui-sidebar-link_active" aria-haspopup="true" aria-expanded="false">
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: `type`

File: packages/oui-sidebar/README.md [hunk 8]
```diff
@@ -0,0 +1,689 @@
...
+      </li>
+      <li class="oui-sidebar-list__item">
+        <button class="oui-sidebar-link oui-sidebar-link_active" aria-haspopup="true" aria-expanded="true">
+          Category 3
+        </button>
+        <div class="oui-sidebar-menu oui-sidebar-menu_fixed">
+          <header class="oui-sidebar-menu__header">
+            <div class="oui-sidebar-menu__back">
+              <button type="button" class="oui-button oui-button_link oui-button_icon-left">
+                <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: Accessibility: how do blind people know what this button does ?

File: packages/oui-sidebar/README.md [hunk 9]
```diff
@@ -0,0 +1,689 @@
...
+          Category 1
+        </button>
+      </li>
+      <li class="oui-sidebar-list__item" aria-haspopup="true" aria-expanded="false">
+        <button type="button" class="oui-sidebar-link">
+          Category 2
+        </button>
+      </li>
+      <li class="oui-sidebar-list__item">
+        <button class="oui-sidebar-link oui-sidebar-link_active" aria-haspopup="true" aria-expanded="true">
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: `type`

File: packages/oui-sidebar/README.md [hunk 10]
```diff
@@ -0,0 +1,689 @@
...
+          Category 1
+        </button>
+      </li>
+      <li class="oui-sidebar-list__item">
+        <button class="oui-sidebar-link">
+          Category 2
+        </button>
+      </li>
+      <li class="oui-sidebar-list__item">
+        <button class="oui-sidebar-link oui-sidebar-link_active" aria-haspopup="true" aria-expanded="false">
```
Comment (FredericEspiau) [inline packages/oui-sidebar/README.md]: `type`

File: packages/oui-sidebar/README.md (hunk 3 above)
Comment (FredericEspiau) [reply packages/oui-sidebar/README.md]: ref: https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks

File: packages/oui-sidebar/README.md (hunk 5 above)
Comment (FredericEspiau) [reply packages/oui-sidebar/README.md]:
```js
<button type="button" class="oui-button oui-button_link oui-button_icon-left">
	<span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
	<span style="display: none;">Open a new submenu</span>
</button>
```

File: packages/oui-sidebar/README.md (hunk 2 above)
Comment (AxelPeter) [reply packages/oui-sidebar/README.md]: Done

File: packages/oui-sidebar/README.md (hunk 3 above)
Comment (AxelPeter) [reply packages/oui-sidebar/README.md]: So, this means having a `<h1>` in each `<header>` ?

**Pull request merged**